src/objects/inline_keyboard.rs

```diff
@@ -0,0 +1,43 @@
+use std::option::Option;
+
+/// represents an inline keyboard that appears right next to the message it belongs to.
```

Comment:
Please us a capital letter at the beginning of the sentence. When referring to other types do it like `Message`.